english/trapezoid/trapezoid.md

```diff
@@ -0,0 +1,13 @@
+There is a blackboard and a chalk
+
+You draw a perfect trapezoid on the blackboard
+
+The finished trapezoid starts to glow and a hole appears
+
+Mystified, you enter the trapezoid
+
+Do you:
+
+[Lunge for the lighter](lunge-for-the-lighter/lunge-for-the-lighter.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.